tests/server/test_helpers.py

```diff
@@ -35,44 +31,6 @@ class DummyOutputSchema(BaseModel):
     field_3: Any
 
 
-@pytest.mark.parametrize(
-    "unserialized_output, target_serialized_output",
-    [
-        (
-            DummyOutputSchema(
-                field_1=[np.array([[1, 2, 3]])],
-                field_2={"key_1": np.array([[[1, 2, 3]]])},
-                field_3=DummyOutputSchema(field_1=np.array([0])),
-            ),
-            DummyOutputSchema(
-                field_1=[[[1, 2, 3]]],
-                field_2={"key_1": [[[1, 2, 3]]]},
-                field_3=DummyOutputSchema(field_1=[0]),
-            ),
-        )
-    ],
-)
-def test_prep_outputs_for_serialization(unserialized_output, target_serialized_output):
```

Comment:
All this fuss is because I want to reuse `prep_outputs_for_serialization` outside of `server`.
So I:
- change its name so it's more accurate
- change the name in all the places in the code where the function is used
- move the unit tests to appropriate place.